spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,18 @@
+require 'beaker-rspec'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_ca_certs unless ENV['PUPPET_INSTALL_TYPE'] =~ %r{pe}i
+install_module_on(hosts)
```

Comment:
if it's not a multi-node setup, I think this can be shortened to just
```
run_puppet_install_helper
install_module
install_module_dependencies
```